### ISSUE TITLE
Enable history-v2 to everyone

### ISSUE THREAD

**Issue by anandology** [internetarchive/openlibrary#40]
history-v2 feature, if enabled, displays rich messages to history snippet. 

For example, it when enabled it shows "Merged 2 duplicate author records into this master. See details." and not enabled, it shows "merge authors".

IIRC, we tried to enable this feature for everyone and noticed some performance issues and reverted back. We need to revisit that and enable it for everyone without any performance issues.

E.g. view https://openlibrary.org/authors/OL86811A/Ismail_Kadare?m=history logged in vs logged out:

Logged in as admin:
![image](https://github.com/user-attachments/assets/f3cf36f3-ce98-4cdc-8df7-1e27a8c6fc63)

Logged out:
![image](https://github.com/user-attachments/assets/dc9962f9-f78e-4ad3-853d-ebaf60395524)

**Comment by anandology** [comment]
moved to september milestone.

**Comment by jdlrobson** [comment]
I'm not 100% sure what history-v2 is but I've seen mentions of it in the code. Could somebody explain what it is?

**Comment by hornc** [comment]
@mekarpeles Is there still anything left to do for this? I think the described history view is enabled for everyone currently, at least I hope so and we can close this.

**Comment by cdrini** [comment]
In openlibrary.yml, it looks like this is only enabled for admins. Here's what I see:
![image](https://user-images.githubusercontent.com/6251786/59480086-3ecff780-8e2d-11e9-813e-7f287e6c264d.png)
https://openlibrary.org/authors/OL86811A/Ismail_Kadare?m=history

**Comment by guyjeangilles** [comment]
@cdrini Is the screenshot in your comment history-v1 or history-v2? I'm not an admin and I'm getting a similar view.
![history](https://user-images.githubusercontent.com/7256157/68910689-8fd07880-074a-11ea-9aa8-39ea67219e62.png)

**Comment by cdrini** [comment]
I wan't an admin at the time I posted that; this is what it looks like for admins:

![image](https://user-images.githubusercontent.com/6251786/68917047-8fc67d80-0736-11ea-83f4-d20685e8be79.png)

**Comment by tfmorris** [comment]
So it's not enabled for everyone. In addition to fixing whatever performance issues there are, the styling needs to be fixed too (problem front end refactoring regression) - Compare button not styled, margins too narrow (or is that just the screen capture?)

**Comment by cdrini** [comment]
Compare button's always been like that (I think); it's just Ubuntu vs. Windows styling. Margins are just the screenshot.

**Comment by mekarpeles** [comment]
I don't see this as a thing I'm planning to prioritize, so I'm going to unassign myself.

**Comment by mekarpeles** [comment]
I think there may be a PR about this?

**Comment by Freso** [comment]
More than 10 years since filing, more than 2 years since last comment—which mentioned that there might already be a PR for it. Is this still an issue?

**Comment by cdrini** [comment]
Yeah still an issue ; it's currently only enabled for admins -- maybe librarians or all logged in users? Not sure. We don't know what the original perf issues were, and investigating perf issues like this is rather intricate, and we already have a decent amount of performance issues as it is! So spending time on this right now isn't a priority.
